doc/sphinx/source/project/introduction.rst

```diff
@@ -4,176 +4,86 @@
 Introduction
 ############
 
-The nGraph Compiler stack provides industry-standard reference and implementation 
-guidelines for working with various :abbr:`Deep Learning (DL)` (DL) models and 
-optimizing an :abbr:`Artificial Neural Network (ANN)` (often abbreviated :term:`NN`) 
-to run graph-based computations for training, inference, testing, or validation.  
-Because today's NNs make use of many custom-purpose devices (FPGAs, GPUs, CPUs, 
-and custom silicon), having such a standard simplifies what would otherwise be 
-an enormously complex and difficult-to-scale pipeline (:ref:`Figure C <figure-C>`) 
-from "training with your favorite framework using GPUs" (:ref:`Figure D <figure-D>`), 
-to deploying that (now) pre-trained model in a datacenter or production 
-environment, where infrastructure owners or software developers renting anything 
-in a datacenter ought to be mutually concerned with **efficiency per-watt**, to 
-keep costs in check.
-
-So what exactly are the motivations behind the nGraph Compiler stack? 
+Accelerating deep learning performance is critical for further development of Artificial Intelligence (AI). Deep learning models & corresponding data set sizes are increasing rapidly, and deployment of scalable AI solution is challenging. Currently, there are two general approaches to advancing deep learning performance. First approach is to design hardware solutions dedicated to deep learning computation. Exisiting large silicon companies including Intel and deep learning hardware startups are actively developing ASICs (Application Specific Integrated Circuit) for speeding up deep learning performance for training & inference. 
+
+Second approach is to deliever performance via software optimization, and nGraph Compiler stack is our solution to the second apporach. We created nGraph compiler stack not only to provide software acceleration for the upcoming deep learning ASICs but also to unlock massive performance boost for existing hardware targets such as CPUs and GPUs. Additionally, it aims to address the scalability issue with kernel libraries which is a popular solution to accelerate deep learning performance.  
```

Comment:
Typo on "deliver"
 
>  We created nGraph Compiler stack not only to provide software acceleration for upcoming deep learning ASICs but also to unlock massive performance boost for existing hardware targets such as CPUs and GPUs. 

Maybe a better way to phrase this is something like the nGraph approach is to enable people who want to write software/applications *today* (using CPUs and GPUs) that can and will easily benefit from custom ASICs or other datacenter deployments.  The software writer should not be worried  performance at all when they're just trying to get things to work the first time...   Datacenter admins can use nGraph to get more throughput out of their networks, regardless if users are running training or inference, with these customs ASICs.  Or, for totally local custom-built server racks, adding a custom ASIC like NNP provides more secure isolation from networks where sensitive data is being handled.

Reply:
Model once run everywhere, rather than adjust the model for each new device, is a goal.